playgrounds/html/index.html

```diff
@@ -48,14 +50,14 @@
         margin: 10%;
       }
 
-      .searchbox {
+      div[data-ds-theme] .searchbox {
         width: 60%;
         margin: auto;
         margin-top: 10%;
         display: block;
       }
 
-      .searchbox input {
+      div[data-ds-theme] .searchbox input {
```

Comment:
Any idea why this work but not  `.docs-searchbar .searchbox` ?

Reply:
I think that's because we use `div[data-ds-theme='dark']` in the `main.scss`, so we have to use the same selector to override the style :)